features/org.wso2.carbon.identity.data.publisher.application.authentication.server.feature/resources/IsAnalytics-Publisher-wso2event-AuthenticationData.xml.j2

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<!--
+~ Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+~
+~ Licensed under the Apache License, Version 2.0 (the "License");
+~ you may not use this file except in compliance with the License.
+~ You may obtain a copy of the License at
+~
+~ http://www.apache.org/licenses/LICENSE-2.0
+~
+~ Unless required by applicable law or agreed to in writing, software
+~ distributed under the License is distributed on an "AS IS" BASIS,
+~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+~ See the License for the specific language governing permissions and
+~ limitations under the License.
+ -->
+
+ <eventPublisher name="IsAnalytics-Publisher-wso2event-SessionData" statistics="disable" trace="disable" xmlns="http://wso2.org/carbon/eventpublisher">
+   <from streamName="org.wso2.is.analytics.stream.OverallSession" version="1.0.0"/>
+   <mapping customMapping="disable" type="wso2event"/>
+   <to eventAdapterType="wso2event">
+     <property name="username">{{event_publisher.authentication_data.username}}</property>
+     <property name="protocol">thrift</property>
+     <property name="publishingMode">non-blocking</property>
+     <property name="publishTimeout">0</property>
+     <property name="receiverURL">{{event_publisher.authentication_data.receiver_url}}</property>
```

Comment:
Let's have this as publish_url as from IS POV this is the URL that IS will publish the data to